contracts/ColonyNetwork.sol

```diff
@@ -157,22 +162,33 @@ contract ColonyNetwork is ColonyNetworkStorage {
     colonies[colonyCount] = colony;
     _isColony[colony] = true;
 
-    colony.initialiseColony(this);
+    colony.initialiseColony(this, _rewardInverse);
     emit ColonyAdded(colonyCount, etherRouter);
 
     return etherRouter;
   }
 
   function addColonyVersion(uint _version, address _resolver) public
   always
-  auth
+  calledByMetaColony
   {
+    require(currentColonyVersion > 0, "colony-network-not-intialised-cannot-add-colony-version");
+    
     colonyVersionResolver[_version] = _resolver;
     if (_version > currentColonyVersion) {
       currentColonyVersion = _version;
     }
   }
 
+  function initialise(address _resolver) public 
+  auth 
```

Comment:
Is this required? It can only be called once, has to be before anything productive can be done, and will be before the address is public...